src/include/common/string_format.h

```diff
@@ -0,0 +1,99 @@
+#pragma once
```

Comment:
Is this string_format written by you or copied from duckdb? 
I am a little worried about sstream performance

Reply:
I can refactor this to use std::string instead, let me see how that goes.

Reply:
done.